frontend/src/components/Editor.tsx

```diff
@@ -0,0 +1,21 @@
+import { javascript } from '@codemirror/lang-javascript';
+import { vscodeDark } from '@uiw/codemirror-theme-vscode';
+import CodeMirror from '@uiw/react-codemirror';
+
+const Editor = (props: { code: string | undefined; setCode: (arg0: string) => void }) => {
```

Comment:
```suggestion
interface Props {
  code?: string;
  setCode?: (newCode:string) => void;
}
const Editor = (props: Props) => {
```
이렇게 Props를 분리할 수 있을까요? 함수 선언부에 props의 타입을 분리해서 나열하면 가독성이 떨어지는 것 같습니다.

Reply:
저도 같은 생각입니다. 
제 생각엔, setCode 뒤엔 ? 가 필요 없어 보이네요.
추후에 타입 네이밍도 고려해야겠네요

Reply:
감사합니다, 이렇게 분리하겠습니다!